components/eamxx/src/physics/iop_forcing/eamxx_iop_forcing_process_interface.cpp

```diff
@@ -346,7 +346,8 @@ void IOPForcing::run_impl (const double dt)
   const auto Q = get_group_out("tracers").m_monolithic_field->get_view<Pack***>();
 
   // Load data from IOP files, if necessary
-  m_iop_data_manager->read_iop_file_data(timestamp());
+  // TODO: this is using the TS from the beg of the step. Should it use end_of_step_ts() instead?
+  m_iop_data_manager->read_iop_file_data(start_of_step_ts());
```

Comment:
@tcclevenger same comment here: i left as is, but you may want to think about this, and see if it's worth changing.